src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+
+  --star-size: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction:row` is default behavior of elements inside flex so don't write this property unless you need to